tests/AVIATIONWEATHERGOV_API/test_api_data_airport_get.py

```diff
@@ -0,0 +1,140 @@
+# ********RoostGPT********
+
+# Test generated by RoostGPT for test pyapi-aviation-weather_clone_clone using AI Type Open AI and AI Model gpt-5-nano
+# 
+# Test file generated for /api/data/airport_get for http method type GET 
+# RoostTestHash=a5aeacaf12
+# 
+# 
+
+# ********RoostGPT********
+import json
+from pathlib import Path
+
+import pytest
+from requests.exceptions import HTTPError
+
+from validator import SwaggerSchemaValidator
+
+
+# Helper to load endpoint test data from api_data_airport.json
+def load_endpoint_test_data():
+    base_dir = Path(__file__).resolve().parent
+    data_path = base_dir / "api_data_airport.json"
+    with open(data_path, "r", encoding="utf-8") as f:
+        return json.load(f)
+
+
+@pytest.fixture(scope="session")
+def airport_validator():
+    """
+    Swagger schema validator initialized with the OpenAPI spec for the project.
+    Path is resolved relative to this test file.
+    """
+    base_dir = Path(__file__).resolve().parent
+    spec_path = str(base_dir / "api.json")
+    return SwaggerSchemaValidator(spec_path)
+
+
+@pytest.fixture(scope="session")
+def endpoint():
+    """API endpoint under test"""
+    return "/api/data/airport"
+
+
+def _build_params(item: dict) -> dict:
+    """
+    Build query parameters from ENDPOINT_TEST_DATA item.
+    Only include non-empty values to respect optional parameters.
+    """
+    params = {}
+    if item.get("ids"):
+        params["ids"] = item["ids"]
+    if item.get("bbox"):
+        params["bbox"] = item["bbox"]
+    if item.get("format"):
+        params["format"] = item["format"]
+    return params
+
+
+@pytest.mark.smoke
+def test_airport_endpoint_smoke(host, api_client, airport_validator, endpoint):
+    """
+    Smoke test: Happy path with minimal/empty payload (respecting optional params).
+    This validates that the endpoint is reachable and returns a valid 200 response
+    according to the OpenAPI spec.
+    """
+    # Use no params for smoke test to cover minimal request path
+    params = {}
+
```

Comment:
LGTM.
request params are ids, bbox, format . All are optional, so expected smoke test with empty param is a good case.